src/Network/Probe.php

```diff
@@ -1755,6 +1755,9 @@ private static function twitter($uri)
 	public static function getFeedLink(string $url, string $body)
 	{
 		$doc = new DOMDocument();
+		if (empty($body)) {
+			return '';
+		}
```

Comment:
Small formatting suggestion.
```suggestion
		if (empty($body)) {
			return '';
		}

		$doc = new DOMDocument();
```